src/util.rs

```diff
@@ -334,6 +335,36 @@ pub fn format_percent_bar(percent: f32) -> String {
         .collect()
 }
 
+pub fn format_vec_to_bar_graph<T>(content: &[T], min: Option<T>, max: Option<T>) -> String
+where
+    T: Ord + ToPrimitive,
+{
+    let bars = [
+        '\u{2581}', '\u{2582}', '\u{2583}', '\u{2584}', '\u{2585}', '\u{2586}', '\u{2587}',
```

Comment:
Can you add a comment saying what these unicode chars are meant to be?

Reply:
` (x * one eighth block) https://en.wikipedia.org/wiki/Block_Elements`
Added to the file also.